test/mongoose_push_application_test.exs

```diff
@@ -1,25 +1,23 @@
 defmodule MongoosePushApplicationTest do
   use ExUnit.Case
   import MongoosePush.Application
+  import MongoosePush.Pools
   doctest MongoosePush.Application
 
-  test "worker name" do
-    assert :apns_name1_1 == worker_name(:apns, :name1, 1)
-    assert :fcm_name2_12 == worker_name(:fcm, :name2, 12)
-  end
+  setup do
+    # Validate config/text.exs that is need for this test suite
+    apns_pools = Keyword.keys(Application.get_env(:mongoose_push, :apns))
+    4 = length(apns_pools)
```

Comment:
That could be reduced to [:prod1, :prod2, :dev1, :dev2] = apns_pools

But is so minor that you don't have to bother yourself.